app/DoctrineMigrations/Version20181107134447.php

```diff
@@ -27,9 +29,20 @@ public function up(Schema $schema)
             ADD intake_status VARCHAR(50) NOT NULL, 
             ADD contract_signed VARCHAR(50) DEFAULT NULL, 
             ADD surfconext_representative_approved VARCHAR(50) DEFAULT NULL, 
-            ADD privacy_questions_answered VARCHAR(50) DEFAULT NULL, 
             ADD connection_status VARCHAR(50) NOT NULL'
         );
+
+        $this->addSql(
+            sprintf(
+                'UPDATE service SET 
+                service_type = "%s",
+                intake_status = "%s",
+                contract_signed = "%s"',
+                Service::SERVICE_TYPE_NON_INSTITUTE,
```

Comment:
Good point! For the record: 

> because over time your migrations could change

You mean that the Service entity might change right and that the migration becomes unstable in that regard?